src/main/java/seedu/address/logic/parser/AddCommandParser.java

```diff
@@ -35,10 +35,17 @@ public AddCommand parse(String args) throws ParseException {
         ArgumentMultimap argMultimap =
                 ArgumentTokenizer.tokenize(args, PREFIX_NAME, PREFIX_PHONE, PREFIX_EMAIL, PREFIX_ADDRESS, PREFIX_TAG);
 
+        if (!arePrefixesPresent(argMultimap, PREFIX_EMAIL)) {
+            throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT,
+                    AddCommand.MESSAGE_USAGE)); //Need to change the MESSAGE_USAGE.
+        }
+
+        /*
```

Comment:
don't retain commented out code, delete it.
you can always retrieve it from VCS if necessary